README.md

```diff
@@ -26,7 +26,7 @@
 ## Download
 Via F-Droid: [com.simplytranslate_mobile](https://f-droid.org/en/packages/com.simplytranslate_mobile/).
 
-For the most up-to-date versions. Download the most recent **Artifact** from the [Actions](https://github.com/ManeraKai/simplytranslate_mobile/actions) page. They may be slow as they're in debug-mode. You should be logged in before you're able to download.
+For the most up-to-date versions, download the most recent **Artifact** from the [Actions](https://github.com/ManeraKai/simplytranslate_mobile/actions) page. They may be slow as they're in debug-mode. You should be logged in before you're able to download.
```

Comment:
Here it's a one sentence, `For` is like `if`here.

Reply:
I understood it as `Download the most recent artifacts for the most up-to-date versions` or `Download the most recent artifacts if you want the most up-to-date versions`. I can revert this part if you want.

Reply:
Yes, please.

Reply:
Ok, reverted this change.